internals/secrethub/org_init.go

```diff
@@ -39,14 +39,27 @@ func (cmd *OrgInitCommand) Register(r Registerer) {
 func (cmd *OrgInitCommand) Run() error {
 	var err error
 
+	if !cmd.force {
+		confirmed, err := ui.AskYesNo(cmd.io, "Creating an organization is not available on the free plan, would you to like to proceed and start a 14-day trial?", ui.DefaultNo)
```

Comment:
Let's word this positively instead. Also, why would we need to prompt? It's free no strings attached for 14 days, so we can just mention it in the success message right?

Something like:

```
Creating organization...
Organization created. Your 14-day trial starts now. To purchase a subscription for your shared organization workspace, visit https://secrethub.io/pricing
```

Reply:
We could also refer to the getting started on organizations docs instead (these are in the making). These will include more detailed information about working with orgs.

Reply:
Yes, but you do it only once, thereby making it a 'destructive' action. Maybe the user wants to first discuss it with the rest of the team members on whether the moment is right to take some time and start testing the tool in a team setting. So a tiny little prompt won't hurt here, imo.

Reply:
And for the wording: 
```
Creating an organization will start a 14-day trial (no strings attached) of the Team plan. Visit https://secrethub.io/pricing for more information. Would you like to proceed?
```

Reply:
Okay cool, if @SimonBarendse agrees I'm fine with this approach. 

I like how it's relaxed your message is. Well done. A tiny copy nitpick would be to add `free` in there:

`Creating an organization will start a free 14-day trial (no strings attached) of the Team plan. Visit https://secrethub.io/pricing for more information. Would you like to proceed?`

Reply:
Also, the default should be yes. Definitely yes.